static/js/tools/scatter/place_and_type_options.tsx

```diff
@@ -256,6 +256,11 @@ async function loadPlaces(
             `Sorry, ${place.value.enclosingPlace.name} does not contain places of type ` +
               `${childPlaceType}. Try picking another type or place.`
           );
+          document.getElementById("place-list").innerHTML = "";
```

Comment:
same comments as those for map

Reply:
In the scatter, the clearing of the place-type is already done -- hence, reverting this file in this PR.